spaceship/lib/spaceship/connect_api/api_client.rb

```diff
@@ -194,6 +195,11 @@ def handle_response(response)
         return Spaceship::ConnectAPI::Response.new(body: response.body, status: response.status, client: self)
       end
 
+      def handle_401(response)
+        # Spaceship::Client has its own check for 401
+        # Overriding that handling of 401 and letting this handle_errors handle the 401
```

Comment:
Does this comment imply that this method in the superclass should delegate to `handle_errors` ?

Reply:
Ah nope! I should make that clearer 😇 This comment implies that superclass handles errors by default in other clients but we don’t want that so we are overriding it 😊